src/urls.js

```diff
@@ -92,6 +92,10 @@ export function urlApiPersonalToMedia() {
   return `${urlApiBase()}personal_to_media/`
```

Comment:
If this is not needed anymore, you can delete this.

Reply:
Yes.